TCAT/Views/BusIcon.swift

```diff
@@ -76,23 +76,23 @@ class BusIcon: UIView {
         self.backgroundColor = .clear
         
         baseView = UIView(frame: self.frame)
-        baseView.backgroundColor = .tcatBlueColor
+        baseView.backgroundColor = Colors.tcatBlue
         baseView.layer.cornerRadius = type.cornerRadius
         addSubview(baseView)
         
         image = UIImageView(image: UIImage(named: "bus"))
         let constant: CGFloat = type == .directionSmall ? 0.75 : 1.0
         image.frame.size = CGSize(width: image.frame.width * constant, height: image.frame.height * constant)
-        image.tintColor = .white
+        image.tintColor = Colors.white
         image.center.y = baseView.center.y
         image.frame.origin.x = type == .directionSmall ? 8 : 12
         addSubview(image)
         
         label = UILabel(frame: CGRect(x: image.frame.maxX, y: 0, width: frame.width - image.frame.maxX, height: frame.height))
         label.text = "\(number)"
         let fontSize: CGFloat = type == .directionLarge ? 20 : 14
-        label.font = .style(Fonts.System.semibold, size: fontSize)
-        label.textColor = .white
+        label.font = .style(Fonts.SanFrancisco.semibold, size: fontSize)
```

Comment:
maybe im missing smtg here but aren't u supposed to do: `Fonts. SanFrancisco.style(Fonts.SanFrancisco.semibold, size: fontSize)`?

Reply:
The function is UIFont.style, I'm just using type inference